docs/olm-bundle-management.md

```diff
@@ -0,0 +1,1581 @@
+  
+  maintainers:
+    - name: OpenShift Lightspeed Team
+      email: openshift-lightspeed-contact-requests@redhat.com
+  
+  links:
+    - name: Lightspeed Operator
+      url: https://github.com/openshift/lightspeed-operator
+  
+  keywords:
+    - ai
+    - assistant
+    - openshift
+    - llm
+  
+  icon:
+    - base64data: iVBORw0KG...
+      mediatype: image/png
+  
+  replaces: lightspeed-operator.v1.0.5  # For upgrades
+  skips: []  # Versions that can be skipped during upgrade
+```
+
+**Property Reference:**
+
+| Property | Required | Type | Description | Example |
+|----------|----------|------|-------------|---------|
+| `displayName` | Yes | string | Human-readable operator name | `OpenShift Lightspeed Operator` |
+| `description` | Yes | string | Detailed operator description (supports markdown) | Multi-line description |
+| `version` | Yes | string | Semantic version of the operator | `1.0.6` |
+| `maturity` | No | string | Development phase | `alpha`, `beta`, `stable`, `deprecated` |
+| `minKubeVersion` | No | string | Minimum Kubernetes version | `1.28.0` |
+| `provider.name` | Yes | string | Organization providing the operator | `Red Hat, Inc` |
+| `provider.url` | No | string | Provider's website | `https://redhat.com` |
+| `maintainers` | No | array | List of maintainer contacts | `[{name, email}]` |
+| `links` | No | array | Related URLs (docs, source, etc.) | `[{name, url}]` |
+| `keywords` | No | array | Search keywords for OperatorHub | `["ai", "ml"]` |
+| `icon` | No | array | Base64-encoded icon | `[{base64data, mediatype}]` |
+| `replaces` | No | string | Previous version this replaces | `operator.v1.0.5` |
+| `skips` | No | array | Versions skippable during upgrade | `["operator.v1.0.4"]` |
+
+**Upgrade Path Properties:**
+- `replaces`: Defines the upgrade path. Set to the previous version to create a linear upgrade chain
+- `skips`: Advanced feature to skip intermediate versions during upgrade
+- If neither is set, this is treated as a new installation (no upgrade path)
+
+#### Install Modes
+
+Defines where the operator can be installed:
+
+```yaml
+spec:
+  installModes:
+    - type: OwnNamespace      # Install in operator's namespace
+      supported: true
+    - type: SingleNamespace   # Install in one specific namespace
+      supported: false
+    - type: MultiNamespace    # Install watching multiple namespaces
+      supported: false
+    - type: AllNamespaces     # Install watching all namespaces
+      supported: true
+```
+
+**Install Mode Types:**
+
+| Mode | Description | Use Case |
+|------|-------------|----------|
+| `OwnNamespace` | Operator watches its own namespace | Development/testing |
+| `SingleNamespace` | Operator watches one namespace | Namespace isolation |
+| `MultiNamespace` | Operator watches specific namespaces | Multi-tenant with selection |
+| `AllNamespaces` | Operator watches cluster-wide | Cluster-scoped resources (like Lightspeed) |
```

Comment:
Openshift lightspeed supports only OwnNamespace - to watch resources on the namespace its installed on . @blublinsky .
Also are these auto generated files in the docs  folder ?